test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1323,6 +1322,9 @@ def forward(self, arg152_1):
         ),
     )
     @skipIfWindows(msg="Windows don't support quantize.")
+    @unittest.skip(
```

Comment:
Can we delete tests instead of always skipping?  The git history will keep all the details of these tests for when the migration is being done.